hnn_core/viz.py

```diff
@@ -847,9 +880,11 @@ def plot_cell_connectivity(net, conn_idx, src_gid, ax=None, colorbar=True,
     conn_idx : int
         Index of connection to be visualized
         from `net.connectivity`
-    src_gid : int
-        Each cell in a network is uniquely identified by it's "global ID": GID.
-    ax : instance of Axes3D
+    src_gid : int | None
```

Comment:
This is a good description, but after writing so many docstrings it's making me think it will be worthwhile creating some consistency for the arguments that appear super often. Sort of in line with the hnn-core glossary we had talked about, these parameters could start with the boilerplate description, and then more function specific text.

Reply:
I will defer this to a separate PR: https://github.com/jonescompneurolab/hnn-core/issues/387 :)